ui/components/app/home-notification/home-notification.stories.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import HomeNotification from './home-notification.component';
+
+export default {
+  title: 'UI/HomeNotification',
```

Comment:
Title should follow folder structure. I have a PR open that should update all the stories to follow this convention still trying to get it in 😅 
```suggestion
  title: 'Components/App/HomeNotification',
```